macosx/HBTitle.m

```diff
@@ -52,12 +77,15 @@ - (instancetype)initWithTitle:(hb_title_t *)title handle:(hb_handle_t *)handle f
         _hb_title = title;
         _hb_handle = handle;
         _featured = featured;
+        
+        HBMetadata *meta = [[HBMetadata alloc] init];
```

Comment:
you can call directly:
_metadata = [[HBMetadata alloc] initWithMetadata:title->metadata]

Reply:
- [ ] Update statement to pass message directly